content/en/docs/Writing policies/mutate.md

```diff
@@ -44,8 +48,6 @@ A [JSON Patch](http://jsonpatch.com/), implemented as a mutation method called `
 * `replace`
 * `remove`
 
-With Kyverno, the `add` and `replace` have the same behavior (i.e., both operations will add or replace the target element).
```

Comment:
Is this no longer applied?

Reply:
Yes, since we switched JSON path libraries, this statement is no longer true.